codecoverage.py

```diff
@@ -202,6 +206,7 @@ def main():
     parser.add_argument("src_dir", action="store", nargs=nargs, default=default_src_dir, help="Path to the source directory")
     parser.add_argument("branch", action="store", nargs='?', default=default_branch, help="Branch on which jobs ran")
     parser.add_argument("commit", action="store", nargs='?', default=default_commit, help="Commit hash for push")
+    parser.add_argument("output_format", action="store", nargs='?', help="Output format: info or json")
```

Comment:
We can remove this, when run via the command line we always want it to generate the HTML report